packages/aila/src/core/chat/AilaStreamHandler.ts

```diff
@@ -160,7 +159,7 @@ export class AilaStreamHandler {
     }
     log.info("Starting to read from stream");
     try {
-      while (true) {
+      while (!abortController?.signal.aborted) {
```

Comment:
Nice

Reply:
Actually, I realised the controller is optional!

Reply:
Yeah, that was a good spot!